modules/gatsby/src/components/styled/index.js

```diff
@@ -14,101 +52,102 @@ export const BodyContainerFull = styled.div(props => ({
   }
 }));
 
-export const BodyContainerToC = styled.div(props => ({
+
+export const BodyContainerToC = styled('div', ({$theme, ...props}) => ({
```

Comment:
this is exactly how you'd write a base-web styled component. 

import {styled} from 'baseui';
export const MyStyledComponent = styled('div', ({$theme, ...props}) => ({...}));